automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/azure/AzureRuntimeSpec.scala

```diff
@@ -0,0 +1,59 @@
+package org.broadinstitute.dsde.workbench.leonardo.azure
+
+import cats.effect.IO
+import cats.effect.unsafe.implicits.global
+import org.broadinstitute.dsde.workbench.google2.streamUntilDoneOrTimeout
+import org.broadinstitute.dsde.workbench.leonardo.LeonardoConfig.Leonardo.workspaceId
+import org.broadinstitute.dsde.workbench.leonardo.TestUser.{getAuthTokenAndAuthorization, Ron}
+import org.broadinstitute.dsde.workbench.leonardo.{ClusterStatus, LeonardoApiClient, LeonardoTestUtils}
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.prop.TableDrivenPropertyChecks
+import org.scalatest.{DoNotDiscover, ParallelTestExecution, Retries}
+
+import scala.concurrent.duration._
+
+@DoNotDiscover
+class AzureRuntimeSpec
+    extends AnyFlatSpec
+    with LeonardoTestUtils
+    with ParallelTestExecution
+    with TableDrivenPropertyChecks
+    with Retries {
+  implicit val (ronAuthToken, ronAuthorization) = getAuthTokenAndAuthorization(Ron)
+
+  it should "create, get, delete azure runtime" ignore {
+    val runtimeName = randomClusterName
+    val res = LeonardoApiClient.client.use { implicit client =>
+      for {
+        _ <- loggerIO.info(s"AzureRuntimeSpec: About to create runtime")
+//        rat <- Ron.authToken()
```

Comment:
can we remove this line?

Reply:
I left it there just becuz I think in the future when we need to create workspace, we'll need Ron's token